src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -4,107 +4,75 @@ short-title: Supported platforms
 description: The platforms that Flutter supports by platform version.
 ---
 
-## Support Tiers
+## Support tiers
 
 We define three tiers of support for the platforms on
-which built Flutter may be deployed:
+which built Flutter might be deployed:
 
-1. Google-tested platforms
-  are automatically tested at every commit
-  by continuous integration testing.
-1. Best-effort platforms, supported through community
+1. **Supported**, Google-tested platforms
+   are automatically tested at every commit
+   by continuous integration testing.
+1. **Best-effort** platforms, supported through community
    testing, are platforms we believe we support through
    coding practices and ad-hoc testing,
    but rely on the community for testing.
-1. Unsupported platforms, which are platforms that
+1. **Unsupported** platforms, which are platforms that
    might work, but that the development team
```

Comment:
In fact, some of these platforms (e.g. Windows Vista / 32-bit) are those we know *cannot* work. Should we change this text to simply say, "platforms that the development team does not test or support"?